Example-iOS-Swift/WebViewController.swift

```diff
@@ -28,7 +28,7 @@ class WebViewController: UIViewController, WKScriptMessageHandler {
         let script = WKUserScript(source: source, injectionTime: .atDocumentEnd, forMainFrameOnly: false)
         config.userContentController.addUserScript(script)
         config.userContentController.add(self, name: "iosListener")
-        webView = WKWebView(frame: UIScreen.main.bounds, configuration: config)
+        webView = WKWebView(frame: .zero, configuration: config)
```

Comment:
wow this `.zero` so weird, what it do?

Reply:
Well the value passed in here as actually irrelevant as the size is controlled by the parent view controller. `.zero` creates a frame of size 0. I thought making it `.zero` would make it more obvious for peoples.

Reply:
So you can put dot before variables in swift :S